configure.ac

```diff
@@ -1052,9 +1057,37 @@ AS_CASE([$host],
     [AC_CHECK_FUNC([fma], [
       AS_CASE([$target],[x86_64-*-cygwin],[],[AC_DEFINE([HAS_WORKING_FMA])])],
       [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([exp2], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([log2], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([cbrt], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([acosh], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([asinh], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([atanh], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([erf], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([erfc], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([trunc], [], [has_c99_float_ops=false])])
+  AS_IF([$has_c99_float_ops],
+    [AC_CHECK_FUNC([round], [], [has_c99_float_ops=false])])
   AS_IF([$has_c99_float_ops],
     [AC_CHECK_FUNC([copysign], [AC_DEFINE([HAS_C99_FLOAT_OPS])])])])
 
+AS_IF([$has_c99_float_ops],
```

Comment:
This doesn't look right. When `copysign` is the only missing function, `$has_c99_float_ops` is true, but `HAS_C99_FLOAT_OPS` is left undefined.

Reply:
Thanks, good catch, the old `AS_IF` for `copysign` needed updating, too.